interface/templates/interface/submission_result.html

```diff
@@ -6,19 +6,35 @@
         <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
 
         <!-- Bootstrap CSS -->
-        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.min.css" integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T" crossorigin="anonymous">
+        <link rel="stylesheet" href="https://unpkg.com/bootstrap@4.3.1/dist/css/bootstrap.min.css">
+
+        <!-- Holly Javascript -->
+        <script src="http://unpkg.com/jquery@3.2.1/dist/jquery.js"></script>
+        <script src="https://unpkg.com/bootstrap@4.3.1/dist/js/bootstrap.js"></script>
 
         <title>VMCK</title>
+
+        {% include "interface/review_modal.html" %}
     </head>
 
     <body>
         <div class="container">
-            <a class="btn btn-primary" href='{{ homepage_url }}'>Homepage</a>
-            <a class="btn btn-primary" href='{{ submission_list_url }}'>List submissions</a>
-            <p>Submission id: {{ sub.id }}</p>
-            <p>Your score is: {{ sub.score }}/{{ sub.assignment.max_score }}</p>
-            <p>The output is:</p>
-            <pre><samp>{{ sub.output }}</samp></pre>
+            <div class="row mb-2 mt-2">
+                <a class="btn btn-primary mr-1" href='{{ homepage_url }}'>Homepage</a>
+                <a class="btn btn-primary" href='{{ submission_list_url }}'>List submissions</a>
+            </div>
+            <div class="container">
```

Comment:
A `container` within a `container` looks suspicious. I think it's basically a no-op, but it could confuse the layout.